content/applications/finance/fiscal_localizations/united_kingdom.rst

```diff
@@ -227,3 +243,184 @@ use our connector as an alternative payroll solution.
+System Parameters`. Then search for `l10n_uk_hmrc.api_mode`. Select it and switch the value from
+`production` to `test`.
+
+.. _localization/united-kingdom/cis-monthly-returns:
+
+Monthly returns
+---------------
+
+Monthly returns only work for vendor bills and vendor refunds. To submit a complete return to
+|HMRC|, several steps must be followed to report all payments made to subcontractors under the
+scheme during the previous tax month:
+
+- :ref:`localization/united-kingdom/cis-contractor-setup`
+- :ref:`localization/united-kingdom/cis-subcontractor-setup`
+- :ref:`localization/united-kingdom/cis-vendorbills`
+- :ref:`localization/united-kingdom/cis-monthly-return-sending`
+
+.. _localization/united-kingdom/cis-contractor-setup:
+
+Contractor setup
+~~~~~~~~~~~~~~~~
+
+To configure the company |HMRC| information, go to the :guilabel:`Settings` app. In the
+:guilabel:`Companies` section, click :guilabel:`Manage Companies` and select the company. Open the
+:guilabel:`HMRC` tab and configure the information in the :guilabel:`HMRC Credentials` and the
+:guilabel:`Contractor details` sections. All the fields are mandatory.
+
+.. _localization/united-kingdom/cis-subcontractor-setup:
+
+Subcontractor setup
+~~~~~~~~~~~~~~~~~~~
+
+In the :guilabel:`Contacts` app, open the subcontractor's contact form and select the
+:guilabel:`Accounting` tab. In the :guilabel:`HMRC Details` section, enable the
+:guilabel:`Construction Industry Scheme` option to unlock the additional fields.
+
+By default, the :guilabel:`Deduction rate` is set to 30%. To modify it, a
+:guilabel:`Verification Number` is required. The verification number is provided by |HMRC| when
+verifying the contact. Make sure to check the status of the subcontractor and update the
+:guilabel:`Deduction Rate` accordingly.
+
+The :guilabel:`Forename` and :guilabel:`Surname` fields are mandatory if the contact is set to
+:guilabel:`Individual`.
+
+.. _localization/united-kingdom/cis-vendorbills:
+
+Vendor bills
+~~~~~~~~~~~~
+
+Additional |CIS| tax must be applied to labour line items on vendor bills. Material line items
+on vendor bills do not apply. The tax rate corresponds to the subcontractor's
+:guilabel:`deduction rate`: :guilabel:`O% CIS`, :guilabel:`20% CIS` or :guilabel:`30% CIS`. To apply
+this, in the :guilabel:`Invoice Lines` section of the vendor bill, select the appropriate |CIS| tax
+rate in the :guilabel:`Taxes` column of the labour line items.
+
+.. note::
+   - When creating a vendor bill, if the :guilabel:`Construction Industry Scheme` option hasn't been
+     enabled in the :ref:`subcontractor <localization/united-kingdom/cis-subcontractor-setup>`'s
+     :guilabel:`Contact` form, a yellow warning banner appears at the top of the page.
+   - If the |CIS| tax used in the vendor bill does not match the expected |CIS| deduction rate for a
+     :ref:`subcontractor <localization/united-kingdom/cis-subcontractor-setup>`, a yellow warning
+     banner appears at the top of the page.
```

Comment:
Could we maybe group these two to avoid repeating "a yellow warning banner appears at the top of the page" twice?

For example:

A yellow banner appears at the top of the page if:
- A
- B

(just an idea)